src/@chakra-ui/gatsby-plugin/components/Tag/utils.ts

```diff
@@ -0,0 +1,320 @@
+import { cssVar } from "@chakra-ui/react"
+
+// Because the color scheme from the default theme
+// Goes through the `Badge` config, used the variables
+// created from that config.
+export const $badgeBg = cssVar("badge-bg")
+export const $badgeColor = cssVar("badge-color")
+
+export const $tagBoxshadowColor = cssVar("tag-boxshadow-color")
+
+// TODO: Can this get consolidated?
```

Comment:
Good question. I don't have any better idea right now.

However, keeping this in its own file definitely helps, good job on doing that.

Reply:
I can leave that comment in there just in case a lightbulb goes off later! 😅